index.html

```diff
@@ -10,18 +10,19 @@
     <link rel="stylesheet" href="style/review.css">
     <!-- bootstrap css -->
     <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.3.2/css/bootstrap.min.css"> -->
-
+    
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
     <title>VigyBag | Home</title>
-    <link rel="icon" type="image/x-icon" href="/img/logo.png">

```

Comment:
why you remove this ???

Reply:
The dark color text in the logo will not be visible on the dark footer. So I made a light colored logo.